lib/rubocop/cop/style/extra_spacing.rb

```diff
@@ -64,6 +53,19 @@ def autocorrect(range)
 
         private
 
+        def check_tokens(ast, t1, t2)
+          return if t2.type == :tNL
```

Comment:
No, that's a `RuboCop::Token` object. To be honest, I don't know much of that part of the code and the refactoring is a pretty mechanical method extraction.